src/components/NewsCard/NewsCard.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import cx from 'classnames';
+
+import { ImageElement } from 'Image';
+import FormattedDate from 'FormattedDate';
+import Link from 'Link';
+
+type NewsCardProps = {
+  author: string;
+  className?: string;
+  date: string;
+  image: {
+    alt: string;
+    height?: string;
+    sizes?: string;
+    src: string;
+    srcSet?: string;
+    width?: string;
+  };
+  title: string;
+  titleAs?: keyof JSX.IntrinsicElements;
```

Comment:
:+1: like this idea

It would be good to limit this prop to headings only (and probably leave out h1)

Reply:
not sure this it's possible to grab just the headings keys from this, so have just declared more explicitly what is allowed!

agreed would be nice to be able to get a subset of IntrinsicElements, but think this does the job just as well anyway - a good one to know for the future